### DISCUSSION TITLE
Do you support Padding

### DISCUSSION THREAD

**Discussion by ToddCrimson** [9seconds/mtg#244] [Q&A] [answered]
Hi,

Does your implementation support Padding?
As described here ==> https://core.telegram.org/mtproto/mtproto-transports#padded-intermediate

ISPs are blocking the proxy based on packet size.

**Comment by 9seconds** [comment, marked as answer]
Telegram does it by default if TLS mode is used. This is transparent for mtg, it does not inspect frames. But version 1, that supports full protocol including middle proxies shenanigans supports it.
